### ISSUE TITLE
Blockquote/pre emit spurious empty paragraph

### ISSUE THREAD

**Issue by paravoid** [apjanke/ronn-ng#70]
The code currently reads:
```ruby
when 'blockquote'
  […]
  macro 'IP', %w["" 4] if indent
  […]
  macro 'IP', %w["" 0] if indent

when 'pre'
  […]
  macro 'IP', %w["" 4] if indent
  […]
  macro 'IP', %w["" 0] if indent

```

This results into e.g. code blocks ending with an `.IP "" 0`, i.e. the opening of a new paragraph, which is left subsequently empty (also: `.IP "" 0` is also equivalent to `.P`, I believe?). In the following excerpt of the test/code_blocks.roff output, one can see that there is the `.IP`, followed by another `.P`, meaning that there is an empty paragraph between the two code blocks:
```groff
[…]
Some text\.
.IP "" 4
.nf
Some code\.
.fi
.IP "" 0
.P
Here's some <pre> text\.
[…]
```

Finally, it seems that commit f528c259185ad65009724c760ec2ba0cb0d847a1 tried to address the output by using more's `-s`, to strip the extra vertical whitespace. But, I wonder if instead ronn could just not emit those empty paragraphs?

**Comment by apjanke** [comment]
I agree with you here. Thanks for doing the analysis. Scheduling this for fix in 0.11 or before. I'm going to have to do some reading on how exactly `.IP` and `.P` work.
